public/api/conf/1.0/docs/overview.md

```diff
@@ -16,4 +16,4 @@ For more background on the VAT API, see the [MTDfB VAT Guide for Vendors](https:
 
 ### Fraud prevention
 
-It will become compulsory to supply header information for this API from April 2019 to <a href="/api-documentation/docs/fraud-prevention">prevent fraud</a>. We recommend that you include this now.
+It is compulsory to supply fraud prevention header information for this API. You can use the <a href="/api-documentation/docs/api/service/txm-fph-validator-api/1.0">Test Fraud Prevention Headers API</a> to make sure your application can produce correctly formatted fraud prevention headers.   You can use this API during development and as part of your regular quality assurance checks.
```

Comment:
Do relative links like this definitely work? And you're missing the `(opens in a new tab)` text, which can be achieved by adding `target="_blank"` to the `a` tag (e.g. `<a href="..." target="_blank">.....</a>`

Reply:
I believe the link works, the previous link (which linked to something else) was using a relative link.

Reply:
Okay. If you add the extra text and `target` attribute I'll merge this.